app/Console/Commands/InstallCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'council:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simplify installation process';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->intro();
+
+        if (!file_exists('.env')) {
+            exec('mv .env.example .env');
+            $this->line("\r\n.env file successfully created\r\n");
```

Comment:
Why `\r\n`?

Reply:
@m1guelpf  A line break for readability and style purposes? Most likely.

Reply:
exactly, nothing too fancy

Reply:
[Laravel already adds a line break](https://github.com/laravel/framework/blob/5.5/src/Illuminate/Console/Command.php#L444), no need to repeat

Reply:
@m1guelpf Yep, I know ;)

Just wanted to give the custom messages a little more readability by adding some extra space :)